src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,104 @@ docs_area: manage
-- `claim_json_key` is the key of the field to be extracted from the external identity provider's [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken) and mapped to a SQL user. This field is likely to be a [standard claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) such as `email` from which a username is extracted via `principal_regex`.
-- `principal_regex` is a regular expression applied to the field specified by `claim_json_key`. It is used to extract an identifier that can be mapped to a SQL user. For example:
-	- `^([^@]+)@[^@]+$` matches any email address (defined as a string containing one `@` sign) and extracts a username from the string to the left of `@`.
-	- `^(.+)$` maps the claim directly to a principal.
-- `autologin` is a Boolean. When `true`, upon loading the DB Console login page, the browser will automatically initiate the OIDC authentication process by redirecting to `/oidc/v1/login` instead of waiting for the user to log in manually or click the OIDC login button.
-
-## Example (Google OAuth 2.0)
-
-These steps demonstrate how to enable SSO authentication for the DB Console on a [local secure cluster]({% link {{ page.version.version }}/secure-a-cluster.md %}) using [Google's OAuth 2.0](https://developers.google.com/identity/protocols/oauth2) implementation.
+These steps demonstrate how to create an OIDC auth client to use for SSO authentication to the DB Console, using Google Cloud Platform.
 
 1. Open the [Credentials page](https://console.developers.google.com/apis/credentials) for your account at Google APIs.
 
 1. Click **+ CREATE CREDENTIALS** and select **OAuth client ID**. Specify a **web application** from the pulldown menu.
 
-1. Note the *client ID* and *client secret* of the OAuth 2.0 client. You can find these in the dialog that appears after you create the client, or in the details view for the client:
+1. Note the *client ID* and *client secret* of the OAuth 2.0 client, as you will need these values to configure you cluster to use them.
+
+1. Add your cluster's callback URL to the list of **Authorized redirect URIs**. On a local cluster, this will be `https://localhost:8080/oidc/v1/callback`. For a {{ site.data.products.dedicated }} cluster, replace 'localhost' with your cluster's domain. This can be found by opening the DB Console from the will be the URL of the You will later set `server.oidc_authentication.redirect_url` to the same value.
+
+## Configure your cluster to use an OIDC client and provider
+
+**Prerequisites**:
 
-	<img src="{{ 'images/v23.1/google-oidc-client.png' | relative_url }}" alt="Google OAuth 2.0 client details" style="border:1px solid #eee;max-width:100%" />
+You must have the ability to update your cluster settings, which requires a SQL user either the [admin role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role) or the `MODIFYCLUSTERSETTING` role option.
```

Comment:
I believe there is also a MODIFYCLUSTERSETTING system level privilege not just a role grant (https://www.cockroachlabs.com/docs/stable/set-cluster-setting).